.gitignore

```diff
@@ -14,3 +14,6 @@
 /.sass-cache
 Gemfile.lock
 .ruby-version
+
+.idea
```

Comment:
It will be better to add this into [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).